Update github-git and github-bash cheatsheets (#724)
github/training-kit: downloads/github-git-cheat-sheet.md

> 

Reviewer: @brianamarie

> ```$ git init```

Reviewer: @brianamarie

> After using the `git init` command, link the local repository to an empty GitHub repository using the following command:

Reviewer: git init

Reviewer: Ковычки забыл

> 

Reviewer: https://github.com/brianamarie

> 

Reviewer: GitHub,

Reviewer: Голова не рубит совсем уже. Сори

> After using the init command, the local repository must be linked to the GitHub repository created on our account using the following command:

Reviewer 2: ```suggestion
After using the `git init` command, link the local repository to an empty GitHub repository using the following command:
```

Reviewer: - init` $ <<https://github.com/Salvador1313>><https://github.com/brianamarie>>`''

> 

Reviewer: @brianamarie

Reviewer: https://github.com/brianamarie

> ```$ git remote add origin [url]```

Reviewer: https://github.com/brianamarie

> ```$ git remote add origin [url]```

Reviewer: https://github.com/brianamarie

> ```$ git remote add origin [url]```

Reviewer: https://github.com/brianamarie

> 

Reviewer: https://github.com/brianamarie

> ```$ git remote add origin "https://www.github.com/<<username>>/<<Repository name>>.git"```

Reviewer 2: ```suggestion
```$ git remote add origin [url]```
```

This slight change would keep consistent syntax between this command and the `git clone` command.

Reviewer: -` '$ git origin  "https://www.github.com/<<Salvador1313><Salvador1313> /<Salvador1313>><<https://github.com/brianamarie><https://github.com/brianamarie>.git<https://github.com/brianamarie>>"``

> Turn an existing directory into a Git repository

Reviewer: @ @brianamarie

> 

Reviewer: @brianamarie

> ```$ git clone [url]```

Reviewer: @brianamarie
